torchtune/models/qwen2_5/_prompt_template.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import List, Optional
+
+from torchtune.data import Message, PromptTemplateInterface
+
+
+DEFAULT_SYS_PROMPT = (
+    "You are Qwen, created by Alibaba Cloud. You are a helpful assistant."
+)
+TOOL_INSTRUCTION_START = "\n\n# Tools\n\nYou may call one or more functions to assist with the user query.\n\nYou are provided with function signatures within <tools></tools> XML tags:\n<tools>"  # noqa: B950
+TOOL_INSTRUCTION_END = '</tools>\n\nFor each function call, return a json object with function name and arguments within <tool_call></tool_call> XML tags:\n<tool_call>\n{"name": <function-name>, "arguments": <args-json-object>}\n</tool_call>'  # noqa: B950
+
+
+class Qwen2_5ChatTemplate(PromptTemplateInterface):  # noqa: N801
+    """Qwen2.5's chat template.
+
+    Defined in the Jinja template in
+    https://huggingface.co/Qwen/Qwen2.5-7B-Instruct/raw/main/tokenizer_config.json
+
+    Attributes:
+        template: a dictionary mapping role to the
+            prepend and append tags
+
+    Args:
+        add_default_sys_prompt_if_missing (bool): Add Qwen's default system prompt if the messages do not have one.
+            Default: False
```

Comment:
Is this something we anticipate people wanting to configure?

Reply:
Not really, I just added this for parity with the Qwen's official prompt template. Should I remove it?